contrib/zpages/src/main/java/io/opencensus/contrib/zpages/Style.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.contrib.zpages;
+
+final class Style {
+  private Style() {}
+
+  static final String STYLE =
```

Comment:
It seems the `final` keyword here causes some performance issue. Could you update this to `static String style` and update other references?

Reply:
So I'm only removing keyword `final` from line 22? Got it!